### ISSUE TITLE
Requesting sync with MyAnimeList (MAL)

### ISSUE THREAD

**Issue by vvirtues** [justchokingaround/jerry#43]
I already use MAL, it probably isn't too difficult to switch, but can you please add support for syncing to MAL?

**Comment by vvirtues** [comment]
still on the table?

**Comment by justchokingaround** [comment]
yes, but low priority, since it'd take considerable effort to implement

**Comment by Corbett000** [comment]
I just started using this project the other day and was having the same issue. My solution was to utilize MALSync as a browser extension, but I can see this not being a great solution for everyone. I just made sure to sync AniList with MAL before setting AniList as the master as I believe MALSync can overwrite parts of your lists if synced improperly.
